python/mujincontrollerclient/realtimerobotclient.py

```diff
@@ -684,7 +684,7 @@ def SaveScene(self, timeout=10, **kwargs):
             filename (str): e.g. /tmp/testscene.mujin.dae, if not specified, it will be saved with an auto-generated filename
             preserveexternalrefs (bool): If True, any bodies that are currently being externally referenced from the environment will be saved as external references.
             externalref (str): If '*', then each of the objects will be saved as externally referencing their original filename. Otherwise will force saving specific bodies as external references.
-            saveclone: If 1, will save the scenes for all the cloned environments
+            saveclone: (DEPRECATED) If 1, will save the scenes for all the cloned environments
```

Comment:
`saveclone` used by binpickingtask3.py

Reply:
My bad. It's not deprecated but should live in `binpickingclient.py` instead. Removed this change for now, will take care of it when the clients are generated.